yarn-project/pxe/src/synchronizer/synchronizer.test.ts

```diff
@@ -31,7 +33,8 @@ describe('Synchronizer', () => {
 
     aztecNode = mock<AztecNode>();
     database = new KVPxeDatabase(await AztecLmdbStore.create(EthAddress.random()));
-    synchronizer = new TestSynchronizer(aztecNode, database);
+    jobQueue = new SerialQueue();
+    synchronizer = new TestSynchronizer(aztecNode, database, jobQueue);
```

Comment:
the job queue is only used when `synchronizer.start` is used. This doesn't happen in this test, nor does it happen in the pxe_service tests.